src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java

```diff
@@ -249,6 +249,31 @@ public List<MedicalWard> getMedicalsWard(char wardId, boolean stripeEmpty) throw
 		return medicalWards;
 	}
 
+	/**
+	 * Gets all the {@link Medical}s associated to Specified {@link Ward} and
+	 * Specified {@link Medical}.
+	 * 
+	 * @param wardId      the ward id.
+	 * @param medId       the medical id.
+	 * @param stripeEmpty - if {@code true}, stripes the empty lots
+	 * @return the retrieved medicals.
+	 * @throws OHServiceException if an error occurs during the medical retrieving.
+	 */
```

Comment:
Does this suggestion apply to the case?

```suggestion
	/**
	 * Get the list of {@link Medical} associated to the specified {@link Ward} and
	 * the specified {@code id}, divided by {@link id}.
	 * 
	 * @param wardId the ward id.
	 * @param medId the medical id.
	 * @param stripeEmpty if {@code true}, stripes the empty lots
	 * @return the requested medical, divided by lots
	 * @throws OHServiceException if an error occurs during the medical retrieving.
	 */
```